lib/offsite_payments/integrations/gestpay.rb

```diff
@@ -189,6 +189,8 @@ def decrypt_data(shop_login, encrypted_string)
           response = ssl_get(Gestpay.service_url, decryption_query_string(shop_login, encrypted_string))
           encoded_response = parse_response(response)
           parse_delimited_string(encoded_response, DELIMITER, true)
+        rescue GestpayEncryptionResponseError => e
```

Comment:
Where is this exception class defined? Can we change it, and make it inherit from an exception class that we do already handle?